commit 1a1377b0721a7d855fed6ea67fb79853b1771e4a
Author:     Yuhong Wen <yuhongw@nvidia.com>
AuthorDate: Thu Apr 4 11:16:01 2024 -0400
Commit:     GitHub <noreply@github.com>
CommitDate: Thu Apr 4 11:16:01 2024 -0400

    cherry-pick the fix_submit_job_auth_failure to 2.4 branch. (#2466)
---
 nvflare/private/fed/server/server_engine.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index 95ba1d1f..a0d141e7 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -760,7 +760,7 @@ class ServerEngine(ServerEngineInternalSpec):
     def _make_message_for_check_resource(self, job, resource_requirements, fl_ctx):
         request = Message(topic=TrainingTopic.CHECK_RESOURCE, body=resource_requirements)
         request.set_header(RequestHeader.JOB_ID, job.job_id)
-        request.set_header(RequestHeader.REQUIRE_AUTHZ, "true")
+        request.set_header(RequestHeader.REQUIRE_AUTHZ, "false")
         request.set_header(RequestHeader.ADMIN_COMMAND, AdminCommandNames.CHECK_RESOURCES)
 
         set_message_security_data(request, job, fl_ctx)
